pkg/telemetry/context.go

```diff
@@ -0,0 +1,53 @@
+package telemetry
+
+import (
+	"context"
+	"strings"
+
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"google.golang.org/grpc"
+)
+
+type rpcContextName string
+
+const (
+	methodContextName  rpcContextName = "method"
+	serviceContextName rpcContextName = "service"
+)
+
+func saveMethodServiceInContext(ctx context.Context, method string, service string) context.Context {
```

Comment:
No need for this function. This is the purpose of the context setter `ContextWithRPCInfo`